addons/web/static/tests/views/search_panel_tests.js

```diff
@@ -617,6 +617,79 @@ QUnit.module('Views', {
         kanban.destroy();
     });
 
+    QUnit.test('category with parent_field and with included child record', async function (assert) {
+        assert.expect(16);
+
+        this.data.company.records.push({id: 40, name: 'child company 1', parent_id: 5});
+        this.data.company.records.push({id: 41, name: 'child company 2', parent_id: 5});
+        this.data.partner.records[0].company_id = 40;
+
+        var kanban = await createView({
+            View: KanbanView,
+            model: 'partner',
+            data: this.data,
+            mockRPC: function (route, args) {
+                if (route === '/web/dataset/search_read') {
+                    assert.step(JSON.stringify(args.domain));
+                    // 'child_of' domain is not evaluated by the domain.js so convert it to 'in' domain and only first level parent supported.
+                    var domain = args.domain && args.domain[0];
```

Comment:
you may simply check that the domain is what you expect (i.e. child_of as operator), and return whatever you want just to prevent it from crashing (thus no longer making assertions on the number of kanban records)

Reply:
@aab-odoo , Done